cluster/examples/kubernetes/ceph/cluster-minimal.yaml

```diff
@@ -24,5 +24,8 @@ spec:
     allowMultiplePerNode: false
   dashboard:
     enabled: true
+  monitoring:
+    enable: true  # requires Prometheus to be pre-installed
```

Comment:
I did think about following the pattern. But,
`enabled: true` would mean it is (already) enabled, but `enable: true` would mean we need to enable it?

Reply:
Think of it as the "desired" state. This is why it should be an attribute "enabled" instead of an action "enable". It's the operator's responsibility to make sure that it's in the desired state. This means:
- `enabled: true`: The operator ensures the ceph-mixins are installed
- `enabled: false`: The operator ensures the ceph-mixins are not installed

Reply:
ack.